README.md

```diff
@@ -2,3 +2,16 @@
 
 Summary reports of login.gov activity
 
+## Development
+
+### Run Development Server
+
+```bash
+npm install
+```
```

Comment:
This won't run it on its own? Either needs a `postinstall` script in `package.json` (not preferred) or extra `npm run dev` command listed here.

Reply:
whoops, I meant to add `npm run dev` on the second line 🤦